src/components/tables/DataTableCell.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <span v-if="!cellValue && cellValue !== 0">—</span>
-    <span v-else-if="cellType === 'date'">{{ this.transformDate(cellValue) }}</span>
+    <span v-else-if="cellType === 'date'" :data-isoDate="cellValue">{{ this.transformDate(cellValue) }}</span>
```

Comment:
We should stick with kebab-case for attributes.

```suggestion
    <span v-else-if="cellType === 'date'" :data-iso-date="cellValue">{{ this.transformDate(cellValue) }}</span>
```